xbmc/music/tags/TagLoaderTagLib.h

```diff
@@ -49,6 +49,9 @@ class CTagLoaderTagLib : public MUSIC_INFO::IMusicInfoTagLoader
   static int POPMtoXBMC(int popm);
 
 template<typename T>
-   static bool ParseTag(T *tag, EmbeddedArt *art, MUSIC_INFO::CMusicInfoTag& infoTag);
+  static bool ParseTag(T* tag, EmbeddedArt* art, MUSIC_INFO::CMusicInfoTag& infoTag);
+
+private:
+  static int m_totalLen;
```

Comment:
why is this `static`?

Reply:
Because the function is static also, so a non-static member variable isn't allowed.

Reply:
static means its value survives the function scope. Is this what you have in mind and why? Isn’t there s surrounding scope calling the static function that can hold the value and pass it as a parameter to the static function?

Reply:
Yes, I know what it means.  I don't want to re-write stuff just to avoid declaring a single variable static.  It's trivial at best.

Reply:
Well, I disagree with this approach, tbh. It's the most simple approach, but not good design. Let's put all source code of Kodi in one single cpp file. Then we don't even need header files anymore. And no complex build system.